core/iwasm/include/wasm_export.h

```diff
@@ -1407,6 +1407,70 @@ WASM_RUNTIME_API_EXTERN bool
 wasm_runtime_unregister_natives(const char *module_name,
                                 NativeSymbol *native_symbols);
 
+struct WASMGlobalInstance;
+typedef struct WASMGlobalInstance *wasm_global_instance_t;
+/**
+ * Get an export global
+ *
+ * @param exec_env the execution environment to retrieve
+ * @param module_name the export module name
+ * @param name the export global name
+ *
+ * @return the export global
+ *
+ */
+wasm_global_instance_t
+wasm_runtime_get_export_global_instance(wasm_exec_env_t exec_env,
+                                        const char *name);
+
+/**
+ * Get the kind of a global instance
+ *
+ * @param export_global the global instance
+ *
+ * @return the kind of the global instance
+ *
+ */
+wasm_valkind_t
+wasm_runtime_global_instance_get_kind(wasm_global_instance_t export_global);
+
+/**
+ * Get the mutability of a global instance
+ *
+ * @param export_global the global instance
+ *
+ * @return true if mutable, false otherwise
+ *
+ */
+bool
+wasm_runtime_global_instance_get_mutable(wasm_global_instance_t export_global);
+
```

Comment:
how about adding one new function to get global type from a global instance and reusing `wasm_global_type_get_valkind` and `wasm_global_type_get_mutable`? Then we can save one new adding

Reply:
@lum1n0us just to make sure I'm understanding, you're saying to create a new struct type to hold the mutable and valkind members, which we could expose in wasm_export.h? If so, what do you think would be a good name for that?

Reply:
I am thinking aobut `wasm_global_type_t`

Reply:
I guess we could just move this definition into wasm_export.h (currently it's in wasm.h):

```
typedef struct WASMGlobalType {
    uint8 val_type;
    bool is_mutable;
} WASMGlobalType;
```

Reply:
I think @wenyongh's idea below should work, and also addresses this issue, so I will try that.